ngi_pipeline/engines/piper_ngi/utils.py

```diff
@@ -41,6 +41,35 @@ def launch_piper_job(command_line, project, log_file_path=None):
         log_process_non_blocking(popen_object.stderr, LOG.warn)
     return popen_object
 
+def spawn_child_md5(project):
+    """Spawns deatached child processes that calculates 
+    md5sum on the bam files in dir 05 and .genomic.vcf files
+    in dir 07.
+    """
+    path=locate_project(project, 'ANALYSIS')
+    fileList05=glob.glob("{}/piper_ngi/05_processed_alignments/*".format(path))
+    fileList07=glob.glob("{}/piper_ngi/07_variant_calls/*".format(path))
+
+    for item in fileList05:
+        if item.endswith((".bam",".bai",".metrics")):
```

Comment:
Same comment as above, @vezzi and @parlundin both think it makes more sense to have everything calculated on scratch. I tend to agree. Even if it makes more files to copy back, we can only guarantee that the files are correct on scratch.